apps/vscode/src/app/telemetry.ts

```diff
@@ -28,10 +28,13 @@ export function initTelemetry(context: ExtensionContext, store: Store) {
   const enableTelemetry = 'enableTelemetry';
   const configurationSection = VSCodeStorage.configurationSection;
   const telemetrySetting = `${configurationSection}.${enableTelemetry}`;
+  const configuration = workspace.getConfiguration(
+    VSCodeStorage.configurationSection
+  );
 
   disposer = workspace.onDidChangeConfiguration(e => {
     if (e.affectsConfiguration(telemetrySetting)) {
-      if (store.get(enableTelemetry, true)) {
```

Comment:
This is the line that has tracking default to true. Remove it will make telemetry opt in rather than opt out